test/hydro/inner_friction_is_proportional_to_velocity_gradient_test.py

```diff
@@ -0,0 +1,104 @@
+from collections import namedtuple
+from pytest import approx, fixture, raises
+from symplyphysics import errors, units, convert_to, Quantity, SI
+from symplyphysics.laws.hydro import inner_friction_is_proportional_to_velocity_gradient as newtons_law
+
+# Description
+## Water (dynamic viscosity of which is 8.9e-4 Pa*s) is flowing steadily on top of a flat solid plate.
+## At the site of contact with the plate its speed is 0, and at the top it is 0.1 m/s. The flow is 5 cm thick.
+## Assert that the shear stress in the water amounts to 1.78e-3 Pa.
```

Comment:
What layer of water are we talking about?

Reply:
![image](https://github.com/blackyblack/symplyphysics/assets/115647936/4426b775-a6b9-4575-883e-085cfd9117c7)
I'm attempting to put this behaviour of the laminar flow of a fluid into words

Reply:
I think shear stress depends on the layer it's applied to. That's why I ask to clarify, which layer you are calculating for.

Reply:
My original intention was the whole cross-section of the water flow (as in the picture).

Reply:
I'm looking to clarify this `Assert that the shear stress in the water amounts to`. Which part of the water are we talking? I assume you are calculating the shear stress at the highest point of water, which is in contact with plate. But you should write it in the description.